src/App.js

```diff
@@ -34,11 +37,20 @@ function App() {
       {isLoading ? (
         <p>Loading...</p>
       ) : (
-        <ConferenceList conferences={conferences} />
+      <Router>
+        <Switch>
+          <Route exact path="/">
+            <ConferenceList conferences={conferences} />
+          </Route>
+          <Route path="/:id">
+            <ConferenceDetails conferences={conferences} />
+          </Route>
```

Comment:
If you guys like you can add a redirect at the end to account for any other url people might type in.https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/Redirect.md

Reply:
Do you mean if someone types in the wrong URL they will be redirected back to the homepage?

Reply:
Well currently they won't be, but if that is your desired outcome then yes you'd need a redirect.